packages/block-library/src/cover/edit.js

```diff
@@ -220,19 +285,18 @@ class CoverEdit extends Component {
 									type="number"
 									id={ inputId }
 									onChange={ ( event ) => {
-										let coverMinHeight = parseInt( event.target.value, 10 );
-										this.setState( { coverMinHeight } );
+										const coverMinHeight = parseInt( event.target.value, 10 );
 										if ( isNaN( coverMinHeight ) ) {
-											// Set cover min height to default size and input box to empty string
-											this.setState( { coverMinHeight: COVER_DEFAULT_HEIGHT } );
-											coverMinHeight = COVER_DEFAULT_HEIGHT;
-										} else if ( coverMinHeight < COVER_MIN_HEIGHT ) {
-											// Set cover min height to minimum size
-											coverMinHeight = COVER_MIN_HEIGHT;
+											setAttributes( { minHeight: undefined } );
+											return;
 										}
-										setAttributes( { minHeight: coverMinHeight } );
+										setAttributes( {
+											minHeight: coverMinHeight > COVER_MIN_HEIGHT ?
+												coverMinHeight :
+												COVER_MIN_HEIGHT,
+										} );
 									} }
-									value={ this.state.coverMinHeight || minHeight }
+									value={ temporaryMinHeight || minHeight }
```

Comment:
the input value is now controlled from one size only, the resizer, if you set a value here, then trigger  `onResizeStopEvent` the value you set here will be neglected and the height will be the last height set by resizing.

Reply:
I'm not sure if I understood, in the input field we can set the height for some value. If later a resizing happens the value set by the resizer is used as expected. After using the resizing the user can use again the input field to other value, the last used interaction is the one that is stored as in all other cases.

Reply:
to better reproduce this issue on your side, try these steps:
1- resize an image using the handles to something like 1000px.
2- resizing the image in the input to something like 500px, the image will indeed resize.
3- attempt to resize from the handles again, the image will jump to 1000px directly, and the input will update to that value directly, therefore neglecting the previous value 500px.

Reply:
Hi @senadir thank you for detailing the problem 👍  I'm not being able to reproduce it. I think it may have been fixed during the last updates :)